.codeclimate.yml

```diff
@@ -63,6 +63,7 @@ plugins:
     enabled: true
     exclude_patterns:
     - 'spec/**/*'
+    - 'db/**/*'
```

Comment:
We should stay in sync with the config here:
https://github.com/18F/identity-idp/blob/master/.codeclimate.yml
It excludes db/migrate/* and db/schema.rb which should address the same issues. This is what I have been instructed to do in the past.

Reply:
@solipet @blacktm FYI

Reply:
@achapm On the that [config](https://github.com/18F/identity-idp/blob/master/.codeclimate.yml) I see these additional items to exclude:
-  'node_modules/**/*'
- 'db/schema.rb'
- 'app/forms/password_form.rb'
- 'lib/user_flow_exporter.rb'
- 'lib/rspec/formatters/user_flow_formatter.rb'

I'll update those in this PR. What about db/migrate/* ? It is not listed there.